src/EVM/SymExec.hs

```diff
@@ -308,9 +311,12 @@ interpret fetcher maxIter askSmtIters heuristic vm =
                     performQuery
                   -- otherwise just try both branches and don't ask the solver
```

Comment:
can we update the comment here?

Reply:
Ooops, thanks, good point! Fixed in 2d29afa4542dedf66ac3afe9863580695b89cfad